optaplanner-build-parent/pom.xml

```diff
@@ -356,6 +357,12 @@
   <build>
     <pluginManagement>
       <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-dependency-plugin</artifactId>
+          <!-- The first version that supports analysis of classes compiled by Java 11. -->
+          <version>3.1.2</version>
```

Comment:
Overwrite property `<version.dependency.plugin>` instead, just like we have this property:


    <version.surefire.plugin>2.22.2</version.surefire.plugin>

Reply:
Good catch, thanks!